README.md

```diff
@@ -1,100 +1,64 @@
-```
-> You can add a `-p <install-prefix>` to choose where to install miniconda. By default, it will install it to `$HOME/miniconda3`.
-
-### Setting up conda for bash
-Copy and paste the following into a terminal running the `bash` shell:
-```bash
-echo -e "\n\n# Adding miniconda3 to PATH" >> $HOME/.bashrc && \
-echo -e "export PATH=$HOME/miniconda3/bin:\$PATH" >> $HOME/.bashrc && \
-source $HOME/.bashrc && \
-which conda
-```
-> *For more information: [bash installation](https://conda.io/docs/user-guide/install/linux.html "Reference to bash conda install")*
-
-### Setting up conda for tcsh
-Copy and paste the following into a terminal running the `tcsh` shell:
-```tcsh
-echo  "\n\n# Setting up miniconda3 for tcsh" >> $HOME/.cshrc && \
-echo  "source $HOME/miniconda3/etc/profile.d/conda.csh > /dev/null" >> $HOME/.cshrc && \
-source $HOME/.cshrc && \
-which conda
-```
-> *For more information: [tcsh installation](https://github.com/ESMValGroup/ESMValTool/issues/301 "Reference to tcsh conda install")*
-
-### Creating an isolated conda environment
-Run the following commands to create a self-contained dev environment for CSM-CameraModel (type `y` to confirm creation):
-```bash
-conda create -n csmdev -c usgs-astrogeology cmake libcsm
-```
-> *For more information: [conda environments](https://conda.io/docs/user-guide/tasks/manage-environments.html)*
-
-### Activating the environment
-After creating the `csmdev` environment and installing cmake and libcsm into it, we need to activate it. Right now, cmake, libcsm, and their dependencies are isolated to a conda environment and we need to tell conda that we want to use it. The activation command depends on your shell.
-* **bash**: `source activate csmdev`
-* **tcsh**: `conda activate csmdev`
-> *You can add these to the end of your $HOME/.bashrc or $HOME/.cshrc if you want the `csmdev` environment to be active in every new terminal.*
+Community Sensor Model (CSM) compliant sensor models created by USGS Astrogeology
+Science Center.
 
-## Building CSM-CameraModel
-After you've set up conda, you can build CSM-CameraModel:
-
-1. Fork `USGS-Astrogeology/CSM-CameraModel` if you don't already have a fork.
-2. Clone your fork of `CSM-CameraModel` *with `--recursive` option to get the gtest submodule*.
-```bash
-git clone --recursive git@github.com:<your-username>/CSM-CameraModel.git
-cd CSM-CameraModel
-git remote add upstream git@github.com:USGS-Astrogeology/CSM-CameraModel.git
-```
-3. Sync your fork with `upstream` and ensure the gtest submodule is init'd if your fork is old.
-```bash
-git pull upstream master
-git submodule update --init --recursive
-git push -u origin master
-```
-4. `mkdir build && cd build`
-5. `cmake .. && make`
-6. `ctest`
+CSM-CameraModel contains two different sensor models. The first, is a generic
+framing camera model written from scratch. The second is a generic line scan
+camera model based on code from BAE Systems Information and Electronic Systems
+Integration Inc.
```

Comment:
Inc. (see LICENSE file).

Reply:
Not sure how to handle our public domain and BAE's BSD_like license. Should we put the BAE license next to the BAE code? Here is the BSD license we need to put somewhere: https://www.dropbox.com/s/nfilxbcqkdmh5hr/license.txt?dl=0

Reply:
Are you sure? I copied their name straight from the comment block they put at the top of the source files.